src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -50,6 +50,7 @@
 import com.gargoylesoftware.htmlunit.util.WebResponseWrapper;
 import com.gargoylesoftware.htmlunit.xml.XmlPage;
 import com.google.common.net.HttpHeaders;
+import com.trilead.ssh2.crypto.Base64;
```

Comment:
Why not use [the standard version](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html)?

Reply:
Miss-import 😞